Linkedlst.rb

```diff
@@ -0,0 +1,95 @@
+class Node
+
+    attr_accessor :value, :next_node
+
+    def initialize val
+        @value = val
+        @next_node = next_node
+        puts  value.to_s 
+    end
+end
+
+class LinkedList
+
+    def initialize val
```

Comment:
Here again, you don't need an argument.  I should be able to create a LinkedList without a node.